samples/openapi3/client/3_0_3_unit_test/python/unit_test_api/api_client.py

```diff
@@ -297,7 +297,7 @@ def _serialize_form(
         prefix_separator_iterator: typing.Optional[PrefixSeparatorIterator] = None
     ) -> str:
         if prefix_separator_iterator is None:
-            prefix_separator_iterator = PrefixSeparatorIterator('?', '&')
```

Comment:
In the caller, please set percent_encode to True:
```
    def __serialize_application_x_www_form_data(
        self, in_data: typing.Any
    ) -> SerializedRequestBody:
        """
        POST submission of form data in body
        """
        if not isinstance(in_data, frozendict.frozendict):
            raise ValueError(
                f'Unable to serialize {in_data} to application/x-www-form-urlencoded because it is not a dict of data')
        cast_in_data = self.__json_encoder.default(in_data)
        value = self._serialize_form(cast_in_data, name='', explode=True, percent_encode=False)
        return dict(body=value)
```
Which is required per https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST